sys/riscv/riscv/support.S

```diff
@@ -40,15 +40,51 @@ __FBSDID("$FreeBSD$");
 
 #include "assym.inc"
 
+/*
+ * Hybrid kernels use .cap loads and stores whereas purecap kernels use
+ * capmode loads and stores.
+ */
```

Comment:
Should these and the ones in copyout.S move to a .inc or .h file?

Reply:
They aren't generically useful (at least, you wouldn't use them in userland I think) and their names are quite generic, so I wouldn't want to put them in something like <machine/asm.h>.  I didn't end up using them anywhere else (and the two files end up using different sets, though copyinout's set is just a subset of these).